source/documentation/other-topics/custom-default-backend.md

```diff
@@ -0,0 +1,78 @@
+
+#### Background
+
+Cloud-Platform created [custom error pages][cloud-platform-custom-error-pages] to be used by the default backend for Nginx ingress-controller. The default backend is the default service that Nginx falls backs to if it cannot route a request successfully. Globally when the service in the Ingress rule does not have active endpoints, this default-backend service will handle the response by serving the cloud-platform custom default error page.
+
+However, some applications don’t want to use the cloud-platform custom default error pages from the Nginx ingress controller but need to be served their own custom error pages. This can be achieved by implementing own [custom default backend][customized-default-backend] in a namespace.
+
+#### Setup
+
+Custom default-backend can be managed inside a namespace by creating a custom default-backend service and deployment of the custom error pages container in your namespace. Configure the name of the service resource created that Nginx should use as the default backend for requests that don’t match any configured ingress rules (hostname and path combinations) using the annotation in your Ingress.
+
+##### Create custom error page
+First create a docker image containing custom HTTP error pages using the [example][ingress-nginx-custom-error-pages] from the ingress-nginx, or [simplified version][cloud-platform-custom-error-pages] of that created by the cloud platform team.
+
+##### Customised default backend
+Using this [custom-default-backend][customized-default-backend] example from ingress-nginx create a service and deployment of the error pages container in your namespace.
+
+```
+$ kubectl -n ${namespace} create -f custom-default-backend.yaml
+service "nginx-errors" created
+deployment.apps "nginx-errors" created
+```
+
+This should have created a Deployment and a Service with the name nginx-errors.
+
+```
+$ kubectl -n ${namespace} get deploy,svc
+NAME                           DESIRED   CURRENT   READY     AGE
+deployment.apps/nginx-errors   1         1         1         10s
+
+NAME                   TYPE        CLUSTER-IP  EXTERNAL-IP   PORT(S)   AGE
+service/nginx-errors   ClusterIP   10.0.0.12   <none>        80/TCP    10s
+```
+
+##### Defining annotations in Ingress file.
+
+Final step is to to use an [Default Backend][default-backend-annotation] annotation in your Ingress, This annotation is of the form nginx.ingress.kubernetes.io/default-backend: <svc name> to specify a custom default backend. This <svc name> is a reference to a service inside of the same namespace in which you are applying this annotation. This annotation overrides the global default backend.
+
+This service will handle the response when the service in the Ingress rule does not have active endpoints. If a default backend annotation is specified on the ingress, the errors will be routed to that annotation's default backend service (instead of the global default backend).
+
+Example Usage:
+```
```

Comment:
Can we show a whole ingress config, with the annotation in place? Or, link to such an example? This would make it easier for users who are not so familiar with kubernetes to understand where this config needs to be.

Reply:
Added an example yaml file